src/containers/ProjectProposals/ProjectProposalsFunding/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React, {FC} from 'react';
+
+import {DocContainer, StepIndicator} from 'components';
+
+import './ProjectProposalsFunding.scss';
+
+const ProjectProposalsFunding: FC = () => {
+  return (
+    <DocContainer className="ProjectProposalsFunding" title="Funding">
+      <h2>How we fund projects?</h2>
+      <StepIndicator
+        className="ProjectProposalsRules__StepIndicator"
+        number={1}
+        text={
+          <>
+            All the members of approved projects will receive <strong>weekly payments</strong> with pay per day of{' '}
+            <strong>2800</strong> coins.
+          </>
+        }
+      />
+      <StepIndicator
+        className="ProjectProposalsRules__StepIndicator"
+        number={2}
+        text={
+          <>
+            The project leader (or manager) has to provide a weekly report of the project’s progress by adding the
+            report file with the name <strong>week-NUMBER.md</strong> (For example, <strong>week-1.md</strong> for the
+            first week, <strong>week-2.md</strong> for the second week report, and so on.) to the{' '}
+            <strong>Weekly Reports</strong> directory of the respective project’s folder in the Project-proposals
```

Comment:
perhaps can link the Project Proposal Repository?

Reply:
Yeah, nice. Done. Updated to google doc as well.